### ISSUE TITLE
Problem when creating a screenshot

### ISSUE THREAD

**Issue by Lukavyi** [avocode/avocode#300]
Describe your problem here…

-----

```
TypeError: undefined is not a function
  at _makeDocumentScreenshot (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/export-panel/lib/stores/export.js:1:2278)
  at t._register (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/export-panel/lib/stores/export.js:1:11929)
  at Object.ID_14 (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/export-panel/lib/stores/export.js:1:459)
  at r.Dispatcher.$Dispatcher_invokeCallback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/flux/lib/Dispatcher.js:220:35)
  at r.Dispatcher.dispatch (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/flux/lib/Dispatcher.js:195:14)
  at r.handleViewAction (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/core/lib/dispatcher.js:1:495)
  at HTMLBodyElement.module.exports.makeDocumentScreenshot (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/@avocode/export-panel/lib/actions/export.js:1:1307)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Avocode.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Avocode.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Avocode.app/Contents/Resources/app.asar/src/window-event-handler.js:178:27)

```

**Comment by rychlis** [comment]
Thanks for the report, this will be fixed in the next release - 2.3.0

**Comment by rychlis** [comment]
We've just released Avocode 2.3.0 which should fix this issue
